pkg/localmetrics/localmetrics.go

```diff
@@ -136,8 +143,11 @@ func UpdateAccountCRMetrics(accountList *awsv1alpha1.AccountList) {
 	MetricTotalAccountPendingVerification.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(pendingVerificationAccountCount))
 	MetricTotalAccountCRsFailed.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(failedAccountCount))
 	MetricTotalAccountCRsReady.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(readyAccountCount))
-	MetricTotalAccountReusedAvailable.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(reusedAccountAvailableCount))
 	MetricTotalAccountReuseFailed.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(reuseAccountFailedCount))
+	for _, account := range accountList.Items {
```

Comment:
Shouldn't we loop thru the idMap slice here? What we're trying to accomplish is having a metric with labels like that below, isn't it?
```json
{
'LegalID-12334546': 12,
'LegalID-XYZZZWX': 7,
}

Reply:
@ninataneja I missed this, but yes this should loop through the dictionary you made.